src/metatable.js

```diff
@@ -1,12 +1,7 @@
-if (typeof module !== 'undefined') {
-    module.exports = function(d3) {
-        return metatable;
-    };
-}
-
-function metatable(options) {
-    var event = d3.dispatch('change', 'rowfocus', 'renameprompt', 'deleteprompt', 'preventprompt');
-    var _renamePrompt, _deletePrompt;
+export default function (options) {
```

Comment:
should `metatable.js` import from d3-set, d3-array, and d3-dispatch, so that it doesn't rely on them being globals if you use the `jsnext:main` entry point?

Reply:
Yep.